webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -29,7 +34,7 @@ const CardItem = ({ content, header, children }) => {
   };
 
   return (
-    <GridItem xl2={3} md={6} lg={5}>
+    <GridItem xl2={3} md={6} lg={5} {...overrideGridProps}>
```

Comment:
just to verify, if `ocerrideGridProps` is e.g. `{'md': 3}` will that override the existing prop?

Reply:
Yes, order matters for JSX spread attributes
```js
// can't be override
 <GridItem  {...overrideGridProps} xl2={3} md={6} lg={5}>
```